parquet/src/main/java/org/apache/iceberg/parquet/ParquetWriter.java

```diff
@@ -96,14 +109,24 @@
     this.model = (ParquetValueWriter<T>) createWriterFunc.apply(parquetSchema);
     this.metricsConfig = metricsConfig;
     this.columnIndexTruncateLength = conf.getInt(COLUMN_INDEX_TRUNCATE_LENGTH, DEFAULT_COLUMN_INDEX_TRUNCATE_LENGTH);
+    this.rowGroupOrdinal = 0;
 
     try {
       this.writer = new ParquetFileWriter(ParquetIO.file(output, conf), parquetSchema,
-         writeMode, rowGroupSize, 0);
+         writeMode, rowGroupSize, 0, columnIndexTruncateLength,
+         // TODO this value is used in current Iceberg. Get from conf instead?
+         ParquetProperties.DEFAULT_STATISTICS_TRUNCATE_LENGTH,
+         // TODO this value is used in current Iceberg. Get from conf instead?
```

Comment:
Do we allow user to set these properties? If yes, we should get them from conf, if no, the current logic should be fine.

Reply:
Sounds good, I'll check if they currently can be set by users.